Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -153,11 +153,96 @@ struct FrameworkBundleAssembler {
             )
         }
     }
+}
+
+extension FrameworkBundleAssembler {
+    struct ResourcesProcessor {
+        private let isVersionedBundle: Bool
+        private let sourceFrameworkBundlePath: TSCAbsolutePath
+        private let sourceFrameworkInfoPlistPath: TSCAbsolutePath
+        private let sourceResourceBundlePath: TSCAbsolutePath?
```

Comment:
How about creating a struct to contain the context? I feel it's strange `isVersionedBundle` is top level. and all variables are bit difficult to read.

```swift
struct ResourceProcessor {
    struct SourceContext {
        let isVersionedBundle: Bool
        let bundlePath: AbsolutePath
        let infoPlistPath: AbsolutePath
        let resourceBundlePath: AbsolutePath
    }
}
```

Reply:
https://github.com/giginet/Scipio/pull/172#discussion_r1915878644

Reply:
isVersionedBundle/bundlePath/infoPlistPath are about a built framework, but resourceBundlePath is about a built resource bundle (that is not separately built by xcbuild and not embedded in a framework), so I'll still keep `framework` and `resource` prefixes.